charts/logstash/templates/deployment.yaml

```diff
@@ -1,30 +1,29 @@
-{{- define "deployments.logstash" -}}
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: logstash
-  namespace: {{ .Values.namespace }}
+  name: {{ .Chart.Name }}
+  namespace: {{ .Values.global.namespace }}
 spec:
-  replicas: 1
+  replicas: {{ .Values.replicaCount }}
   template:
     metadata:
       labels:
-        app: logstash
+        app: {{ template "fullname" . }}
     spec:
       containers:
-      - name: logstash
-        image: logstash:{{ .Values.logstash.image_tag }}
-{{- if .Values.dev_mode }}
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.image.name }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+{{- if or .Values.devMode (and .Values.global .Values.global.devMode) }}
```

Comment:
Same places we're checking for `devMode` like `if .Values.global.devMode` and some places like this: `if or .Values.devMode (and .Values.global .Values.global.devMode)`. Maybe using only the global one?